docs/docsite/rst/intro_patterns.rst

```diff
@@ -99,8 +99,7 @@ And if you want to read the list of hosts from a file, prefix the file name with
 
 Easy enough.  See :doc:`intro_adhoc` and then :doc:`playbooks` for how to apply this knowledge.
 
-.. note:: With the exception of version 1.9, you can use ',' instead of ':' as a host list separator. The ',' is preferred specially when dealing with ranges and ipv6.
-.. note:: As of 2.0 the ';' is deprecated as a host list separator.
+.. note:: You can use ',' instead of ':' as a host list separator. The ',' is preferred specially when dealing with ranges and ipv6.
```

Comment:
We need to leave this here because ':' still works.  Probably anything that has been deprecated for so long but still works deserves to be put on the core team's radar as well to decide what we're doing with it.  For instance, this one is documented as deprecated but using it in a playbook doesn't emit a deprecation warning so it's really unclear what's happening.

Reply:
I only removed the "With the exception of version 1.9..." bit.